kernel/sys/tasks.cpp

```diff
@@ -338,6 +399,10 @@ static void _schedule()
     }
     // reset the time slice because a new task is being scheduled
     _time_slice_remaining = TIME_SLICE_SIZE;
+#ifdef DEBUG
+    px_rs232_print("switching to ");
```

Comment:
Shouldn't this be converted to the new `px_rs232_printf` function?

Reply:
Yeah... but it's just debug code and it doesn't *need* formatting capabilities. Also, the regular print function is much faster than `printf`.

Reply:
Fair enough.